lib/buyers/field.py

```diff
@@ -40,10 +48,35 @@ def to_python(self, value):
         # to save the hash, the same value we see is what is put in the DB.
         if not value.startswith(hasher.algorithm):
             self.salt = hasher.salt()
-            value = make_password(value, salt=self.salt)
+            value = self._hash(value, salt=self.salt)
         return HashedData(value)
 
 
+class ConsistentHashField(HashField):
+
+    """
+    Like the HashField, but is always a consistent value to allow lookups
+    on the field.
+    """
+
+    def get_prep_lookup(self, *args, **kw):
+        return super(CharField, self).get_prep_lookup(*args, **kw)
```

Comment:
this is an atypical use of super and I think it wouldn't be good to rely on this (another dev might delete this code because it looks like a mistake). If the `HashField` constructor is incompatible I suggest making an abstract `HashFieldBase` and inherit each one from that instead.